{{cookiecutter.project_name}}/dev-docker-compose.yml.default

```diff
@@ -0,0 +1,42 @@
+# Configuration for local development
+#
+# See README for setup and usage instructions
+#
+version: '3'
+
+services:
+  postgres:
+    image: postgres
+    ports:
+      - "5433:5433"
+      - "5432:5432"
+    volumes:
+      - "../db:/db"
+      - "../backup:/backup"
```

Comment:
would be good to have the db and backup paths configurable by env var, I have a separate drive for storing all my data for example. Something like `"${PG_DB_DATA:-../db}:/db"` would be cool

Reply:
also, wouldn't those default paths clash with other projects assuming you checkout all projects to a single root dir

Reply:
Yes, this assumes it's checked out to `project/repo`; my standard project structure is
```
project
  repo
  venv
  files
  notes.rst
```
so made sense for me to add `backup` for somewhere to put the sample db. Again, env var would solve this?

Reply:
Also removed build if that's project specific.